### ISSUE TITLE
Chromium 47 - indicator no longer sits in address bar

### ISSUE THREAD

**Issue by alindt** [rauchg/chrome-spdy-indicator#25]
In chromium 47 the indicator is a regular extension button and it no longer sits in the address bar.
Could it be moved back to the address bar?

![spdy_indicator_chromium_47](https://cloud.githubusercontent.com/assets/1379385/11681087/5c851492-9e65-11e5-8062-6d5ae86bdf49.jpg)

**Comment by specious** [comment]
According to an [official announcement](https://groups.google.com/a/chromium.org/forum/#!topic/chromium-extensions/7As9MKhav5E) from the Chromium team, all browser extensions are being switched over to a persistent icon in the toolbar.

**Comment by war59312** [comment]
Yea I saw that temp. on chrome 49 dev. But for now back to normal.

Personally, I hate how devs keeps trying to protect idiots from themselves.

"The reason for this is to protect our users.  We've heard too frequently that many users are unaware of the extensions they have installed, whether this is due to sideloading, installation by phishing, or simply the user forgetting how many and which are installed."

Give me a break. Stop trying to police dumb ass users. Stop trying to fix stuff that ant broken.

No matter what change you make, spyware/adware authors will find a way around it. The price of doing business with allowing extensions in the first place.

**Comment by specious** [comment]
Opening a respectful discussion on the official [Google Chrome Help Forum](https://productforums.google.com/forum/#!forum/chrome) could be beneficial.

**Comment by kav2k** [comment]
This should be closed as it is not controlled by the extension itself.

**Comment by dmitry** [comment]
So it could not be solved anyhow?

**Comment by kav2k** [comment]
@dmitry No, it was Google's decision to change how [Page Actions](https://developer.chrome.com/extensions/pageAction) work. They no longer have the ability to display an icon in the address bar, and it can no longer be optionally shown.

In fact, even official documentation now discourages their use:

> Please consider using a browser action instead, so that users can always interact with your extension.

Announcement and discussion [here](https://groups.google.com/a/chromium.org/d/msg/chromium-extensions/7As9MKhav5E/dNiZDoSCCQAJ) and some after-the-fact analysis [here](https://bugs.chromium.org/p/chromium/issues/detail?id=597657).

**Comment by dmitry** [comment]
@kav2k thanks for the detailed information! :+1:
